Week-3/output-closure/index.js

```diff
@@ -0,0 +1,24 @@
+
+
+function createIncrement() {
+    let count = 0;
+    function increment() {
+        count++;
+    }
+    let message = `Count is ${count}`;
+    function log() {
+        console.log(message);
+    }
+    return [increment, log];
+}
+
+const [increment, log] = createIncrement();
+increment();
+increment();
+increment();
+increment();
+
+log();
+
+
+// Count is 0
```

Comment:
No need to add html again and again, only JS file is enough. You can run your JS file using node